src/Language/Mimsa/Project/Default.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Language.Mimsa.Project.Default
+  ( defaultProject,
+  )
+where
+
+import Language.Mimsa.Types
+
+servers :: [ServerUrl]
+servers = pure (ServerUrl "https://raw.githubusercontent.com/danieljharvey/mimsa-store/master/")
+
+defaultProject :: Project
+defaultProject =
+  Project mempty mempty mempty servers
```

Comment:
A starting set of bindings could be a nice alternative to a Prelude or standard library.